docs/src/assets/scss/components/logo.scss

```diff
@@ -0,0 +1,12 @@
+.logo-component {
+    fill: var(--logo-color);
+}
+
+.logo-title {
+    fill: var(--headings-color);
+}
```

Comment:
```suggestion

```

Do we need to assign a color to the logo title specifically? I don’t recall doing this in the [pr](https://github.com/eslint/eslint/pull/18077/files#diff-06e0f78288fd629ae41ca1ec126819f3f0d4af8d268fd18fc2fb39fbb50c26e6)

Can you check once?

Reply:
yes, saw that PR! in that logo's title was ok on dark mode but in light mode it's color was like this:

![Screenshot 2024-09-17 175000](https://github.com/user-attachments/assets/505f6b26-7564-41f8-9ddf-096571d6ef7b)

Reply:
Cool. got it.